src/utils/variables.js

```diff
@@ -120,6 +120,7 @@ export const baseColors = {
   darkOrange: '#ff8d04',
   chestnutRose: '#cb4c4c',
   darkBlue: '#02050d',
+  desaturatedDarkBlue: '#62688f',
```

Comment:
Pls create a semantic color in utils/themes.js for both lightThemeColorsOld, darkThemeColorsOld (pls ignore the "old" name). Semantic color name is not based on the color description but rather based on color purspose in the app.

Reply:
Done